src/components/FeedbackSurvey.tsx

```diff
@@ -0,0 +1,84 @@
+import {CONST as COMMON_CONST} from 'expensify-common/lib/CONST';
+import React, {useState} from 'react';
+import {View} from 'react-native';
+import useLocalize from '@hooks/useLocalize';
+import useTheme from '@hooks/useTheme';
+import useThemeStyles from '@hooks/useThemeStyles';
+import type {TranslationPaths} from '@src/languages/types';
+import FixedFooter from './FixedFooter';
+import FormAlertWithSubmitButton from './FormAlertWithSubmitButton';
+import SingleOptionSelector from './SingleOptionSelector';
+import Text from './Text';
+
+type FeedbackSurveyProps = {
+    /** Title of the survey */
+    title: string;
+    /** Description of the survey */
+    description: string;
+    /** Callback to be called when the survey is submitted */
+    onSubmit: (reason: Option) => void;
```

Comment:
NAB: please add whitespaces
```suggestion
    /** Title of the survey */
    title: string;
    
    /** Description of the survey */
    description: string;
    
    /** Callback to be called when the survey is submitted */
    onSubmit: (reason: Option) => void;
```